opencog/combo/converter/combo_atomese.cc

```diff
@@ -131,8 +132,7 @@ std::pair<Type, Handle> vertex_2_atom::operator()(const builtin &b) const
 
 std::pair<Type, Handle> vertex_2_atom::operator()(const contin_t& c) const
 {
-	Handle h = createNode(NUMBER_NODE,to_string(c));
-	return std::make_pair(-1, h);
+	return std::make_pair(-1, Handle(createNumberNode(c)));
```

Comment:
BTW, just want to mention that since C++11 one can use `{` `}` instead of `std::make_pair`, which is shorter, though `std::make_pair` has the advantage of being more explicit.

Reply:
Ok. Thanks nil.